src/vs/workbench/parts/files/browser/explorerViewlet.ts

```diff
@@ -180,6 +180,10 @@ export class ExplorerViewlet extends Viewlet {
 		return this.actionRunner;
 	}
 
+	public getOptimalWidth(): number {
+		return this.getExplorerView().getOptimalWidth();
+	}
+
```

Comment:
You need to take the working files view into account too. Note that explorer view can be different when you open no workspace (no folder).